src/pages/juniorPromise/components/TimeSelectionButton.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled';
+import { ButtonCheckIc } from '../../../assets/svgs';
+import { TIME_SELECTION_TITLE } from '../constants/constants';
+
+function TimeSelectionButton() {
+  const buttonValue = '7월 8일 13:00 ~ 13:20';
+  // const buttonValue = null;
+
+  return (
+    <Container>
+      {TIME_SELECTION_TITLE.map((item, index) => (
+        <Wrapper key={index} isActive={buttonValue !== null}>
+          <Title2>{buttonValue ? buttonValue : item.title}</Title2>
+          {buttonValue && (
+            <SvgWrapper>
+              <img src={ButtonCheckIc} alt="check complete button" />
+            </SvgWrapper>
```

Comment:
p1) 저희는 svg파일을 react-componen로 변환해서 사용해주고 있기 때문에 따로 img 태그 안에 작성해주지 않아도 돼요 ! 
custom.d.ts 파일에 보면 svg파일을 리액트 컴포넌트로 변환해주는 타입 모듈이 작성되어 있답니다 !
 svgr세팅을 진행한 것도 위와같은 것을 하기 위해 작성해둔 것이에요.

말 그대로, 피그마에서 추출해 온 svg파일을 index.ts에서 리액트 컴포넌트로 export한 후 그대로 컴포넌트처럼 사용하는 거랍니다!

따라서 다음과같이 사용해보면돼요!
```suggestion
            <ButtonCheckIc />
```

만약 svg에 스타일링을 지정해야 한다면, 다음과 같은 방법으로 사용하면 돼요!
```
const ButtonCheckIcon = styled(ButtoncheckIc)`
 적용해야할 스타일 작성하기!
`
```

Reply:
오..! 이번에 알게된 내용이네요!! `'./ic_check.svg?react'` svg 파일은 ?react 를 붙여 export 하면 되겠네요 감사합니다!